### ISSUE TITLE
Spring @ModelAttribute

### ISSUE THREAD

**Issue by narvik01** [stoicflame/enunciate#412]
Hi.
I have a following endpoint definition in my app:

```
@RequestMapping(value = "/{aUserId}/{bookId}", produces = "application/json", method = RequestMethod.GET)
@TypeHint(Book.class)
public ResponseEntity<Book> getAccount(@ModelAttribute Client client,
                                            @PathVariable("bookId") @NotNull String bookId) {
return null;
}
```

Where Client is filled this way: (@RequestHeader is the spring one, not enunciate)
```
@ControllerAdvice(basePackages = "com.myapp.controllers.api")
public class ClientControllerAdvice {

    @ModelAttribute
    public Client mapClient(@RequestHeader("aUserId") @NotNull @Size(min = 1) String userId,
                                   @RequestHeader("aGroupId") @NotNull String groupId,
                                   @RequestHeader("aSessionId") @NotNull String sessionId) {
        Client client = new Client();
        client.setUserId(userId);
        client.setGroupId(groupId);
        client.setSessionId(sessionId);
        return client;
        }
}
```

Unfortunately, the @RequestHeader does not get picked up by enunciate and it only lists Client fields as query params, not as headers with special names.
For example there is name=sessionId, type=query instead of name=aSessionId, type=header  in the generated docs.
Using enunciate 2.4.0.

**Comment by stoicflame** [comment]
Yeah, so Enunciate doesn't evaluate `@ModelAttribute` yet. 

I'll mark this as an enhancement request and schedule it for 2.5.

**Comment by stoicflame** [comment]
Fixed at f182b95.

**Comment by stoicflame** [comment]
[Enunciate 2.5.0 is now available.](https://github.com/stoicflame/enunciate/releases/tag/v2.5.0)
